server/rpm/Makefile

```diff
@@ -14,11 +14,7 @@ VERSION = $(shell cat ${SERVER}/VERSION)
 
 DISTRO = $(shell uname -r | python -c 'import sys; print(sys.stdin.read().split(".")[-2])')
 
-ifeq (${DISTRO},el7)
-PYTHON = scl enable rh-python36 -- python3
-else
 PYTHON = python3
-endif
 
 RPMSRC = ${HOME}/rpmbuild/SOURCES
```

Comment:
What would really be cool (although technically orthogonal to this PR) would be dependencies to create the `~/rpmbuild` directories we need....

Reply:
Snuck that in. No extra dependencies: instead, when the spec file is created, we create it if necessary, as well as the `rpmbuild/SOURCES` subdir ; `rpmbuild` seems to take care of the rest on its own.

Reply:
Ah; I missed it the first time. Thanks!